cl/opinion_page/views.py

```diff
@@ -872,13 +874,31 @@ async def view_opinion(request: HttpRequest, pk: int, _: str) -> HttpResponse:
     )
     await authorities_context.post_init()
 
+    opinions = cluster.sub_opinions.all()
+
+    # Verify if the cluster has any of the two possible sources with multiple opinions
+    if (
+        SOURCES.HARVARD_CASELAW in cluster.source
+        or SOURCES.COLUMBIA_ARCHIVE in cluster.source
+    ):
```

Comment:
I wonder if this PR can wait until we have ordering across all case law, and, if so, if we can avoid this branch entirely. 

I'd much rather tell our API users to simply order by ordering_key, than to say, "Well, if it's this source or that source, then order this way, otherwise...."

Reply:
At the moment we had only thought about the ordering for Harvard and Columbia but we can add a new action in the command to populate the ordering_key field for all other clusters and automatically set the value in that field every time a new opinion is created. This way we can now order all opinions by ordering_key and get rid of the special cases.

With something like this in the save method:

```python
if self.pk is None and self.ordering_key is None:
	# Add order in new opinions with no defined order value
	last_position = Opinion.objects.filter(cluster=self.cluster).aggregate(models.Max("ordering_key"))["ordering_key__max"]
	self.ordering_key = (last_position or 0) + 1
```

Reply:
Yeah, I think we'll be bummed if we add this sorting key but don't do it well enough to have a simple: "Here's how you sort" comment in the documentation.